docs/architecture/architecture-and-security.md

```diff
@@ -73,29 +78,195 @@ The following image gives a more detailed overview how Landscaper instances are
 
 ![architecture](images/landscaper-instance-creation.png)
 
-For every customer there is a dedicated namespace on the Core-Shoot-Cluster. To get a Landscaper instance a 
+For every customer there is a dedicated namespace on the Core-Shoot-Cluster. To get a Landscaper Instance a 
 user/customer/operator creates a [LandscaperDeployment](../usage/LandscaperDeployments.md) custom resource in this 
 namespace. The LaaS instance in the Core-Shoot-Cluster watches for such custom resources and creates an 
 [Instance](../usage/Instances.md) custom resource for every LandscaperDeployment. For every Instance, the LaaS deploys an 
-[Installation](https://github.com/gardener/landscaper/blob/master/docs/usage/Installations.md) consisting
-of a component which creates
+[Installation](https://github.com/gardener/landscaper/blob/master/docs/usage/Installations.md) for a Landscaper Instance.
 
-- a new shoot custom resource in the Garden-Resource-Cluster-Project for which the Gardener creates the 
-  Resource-Shoot-Cluster
-- a new namespace on one of the Target-Shoot-Clusters for a new Landscaper instance
-- a Landscaper deployment on the Target-Shoot-Cluster in the newly created namespace
+The LaaS furthermore creates two [Targets](https://github.com/gardener/landscaper/blob/master/docs/usage/Targets.md)
+as input for the Installation containing the credentials
 
-The LaaS furthermore creates two [Targets](https://github.com/gardener/landscaper/blob/master/docs/usage/Targets.md) 
-as input for the Installation containing the credentials 
+- for a Target-Shoot-Cluster. These credentials are fetched from one of the ServiceTargetConfig custom resources and 
+  are used to install all required artefacts for the Landscaper Instance on a Target-Shoot-Cluster.
```

Comment:
and the new sidecar service controllers

Reply:
the new sidecar is part of a landscaper instance (one of five sub installations) :-)